src/v2/cookbook/unit-testing-vue-components.md

```diff
@@ -106,7 +106,7 @@ Let's take a look at the component code first:
     <div class="message">
       {{ message }}
     </div>
-    Enter your username: <input v-model="usernane">
+    Enter your username: <input v-model="username">
```

Comment:
Given that this article talks about testing, I'm not 100 % sure whether this typo is intentional or not. It seems, however, that the tests here are about something else (number of characters in the input user name).

Reply:
Yeah, agreed 👍